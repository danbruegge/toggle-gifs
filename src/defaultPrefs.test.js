import defaultPrefs from "./defaultPrefs";

test("defaultPrefs", () => {
  expect(defaultPrefs).toMatchSnapshot();
});
