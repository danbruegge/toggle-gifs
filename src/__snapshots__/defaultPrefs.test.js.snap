// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultPrefs 1`] = `
Object {
  "hoverPauseWhen": 1,
  "hoverPlayOnLoad": false,
  "indicatorStyle": 0,
  "playOnHover": false,
  "shortcutReset": "",
  "shortcutToggle": "",
  "showOverlays": true,
  "supportGifv": true,
}
`;
